### ISSUE TITLE
Add new `## Unreleased` section after replacing it with the just-released tag

### ISSUE THREAD

**Issue by vaind** [getsentry/craft#415]
This is something everyone has to do manually at the moment (or have some other automation do it for them).

**Comment by asottile-sentry** [comment]
could you elaborate on what you're referring to ?

**Comment by asottile-sentry** [comment]
I'm not sure how that would work -- wouldn't that make the top of a just-released changelog say "Unreleased" if it were to add that section at the same time ?  that seems undesirable

**Comment by vaind** [comment]
Yes, it would - that's what the changelog.md looks like anyway - PRs add their items to the `Unreleased` section, then there's a release which just renames it. The first contributor that comes with a PR after a release needs to update their changelog by adding the Unreleased section first.

I mean it's really a first-world-problem of course, just nice to have as it makes life just a bit easier. Also, it helps reduce the chance (of course it doesn't prevent this) of accidentally adding one to an already released section accidentally.

**Comment by asottile-sentry** [comment]
I think what you actually want is a post-release commit to re-add that section -- having it be part of the release will ship that "Unreleased" header to package repositories

**Comment by vaind** [comment]
> I think what you actually want is a post-release commit to re-add that section 

yes exactly

> having it be part of the release will ship that "Unreleased" header to package repositories

Maybe my assumption was wrong then - I thought craft runs the whole release process so it could make a commit **after** it made the release?

**Comment by asottile-sentry** [comment]
the current way craft works is it merges in a commit that's already passed tests -- it could add another commit afterwards but I'm not sure it should without review / passing tests / etc.

**Comment by BYK** [comment]
Why not use the automatic changelog policy instead?

**Comment by BYK** [comment]
Closing as the automatic changelog system is the preferred way for these now and this can also be achieved with the post-publish script.
